fix(website): Update bridge concept (#14665)
taikoxyz/taiko-mono: packages/website/pages/docs/concepts/bridging.mdx

2. Next, we call `eth_getProof`, which will give us a proof that we did indeed send a message on the source chain.
3. Finally, we call `isSignalReceived` on the destination chain's SignalService which essentially just verifies the merkle proof. `isSignalReceived` will look up the block hash you are asserting you had stored a message on the source chain (where you originally sent the message), and with the sibling hashes inside the merkle proof it will rebuild the merkle root, which verifies the signal was included in that merkle root—meaning it was sent.

> And boom! We have sent a cross-chain message. Confusing? Well, leave some feedback please if it is, so we can clear it up but a simple dApp was built during one of our workshops to demonstrate the fundamentals. You can find it [here](https://github.com/adaki2004/MessageService).

Reviewer: I cloned your app into Taiko account, could you use this one instead: https://github.com/taikoxyz/MessageServiceShowCaseApp

> And boom! We have sent a cross-chain message. Confusing? Well, leave some feedback please if it is, so we can clear it up but a simple dApp was built during one of our workshops to demonstrate the fundamentals. You can find it [here](https://github.com/taikoxyz/MessageServiceShowCaseApp).

Reviewer 2: ```suggestion
And boom! We have sent a cross-chain message. If this is confusing, you can also find a simple dApp that was built during one of our workshops to demonstrate the fundamentals. You can find it [here](https://github.com/taikoxyz/MessageServiceShowCaseApp).
```

2. The Bridge locks the Ether, and stores a message by calling `sendSignal(message)` on the SignalService contract
3. The user receives Ether on the destination chain, if they (or another) provide a valid merkle proof that the message was received on the source chain

With the current design there are 2 ways to bridge `Ether`:
1. `Ether` only case: The user interacts directly with the Bridge contract (currently this is the only one supported by our UI) by calling `sendMessage`

> 2. `ERC-XXX` + `Ether` +  case: The user interacts with the `ERCXXXVault` (ERC20, ERC721, ERC1155) because he/she wants to bridge over some tokens, but in case he/she fills the `message.value` field, also `Ether` will be bridged (currently this is not supported by our UI)

Reviewer: I suggest deleting "(currently this is not supported by our UI)" as we will need to delete it soon after UI supports it.
